src/main/java/request/HttpRequestReader.java

```diff
@@ -0,0 +1,13 @@
+package com.td.HttpServer;
+import java.io.IOException;
+
+public class HttpRequestReader implements IRequestReader {
```

Comment:
Yeah, I don't see any value to this class as it is? It just delegates directly to a class/interface that your code still owns.